### ISSUE TITLE
Failed to resolve entry

### ISSUE THREAD

**Issue by ayonli** [christopher-caldwell/mui-inline-suggest#5]
When I tried to import this package into my project which is loaded via Vite, Vite gave me this error:

```
[plugin:vite:import-analysis] Failed to resolve entry for package "@caldwell619/mui-inline-suggest". The package may have incorrect main/module/exports specified in its package.json: Failed to resolve entry for package "@caldwell619/mui-inline-suggest". The package may have incorrect main/module/exports specified in its package.json.
```

I looked up the package.json file, and it includes a `module` property whose value is `dist/index.modern.js`, however, I did not see this file in the installed path, maybe this is where it gets wrong.

Wonderful library BTW, I've tried it on your demo, hope this can be solved ASAP.

**Comment by christopher-caldwell** [comment]
Hey there, sorry about that. I'll take a look. Thanks for making the issue.

**Comment by christopher-caldwell** [comment]
This should be resolved on version 1.3.2. Let me know if it's not!
